src/events/qaMention.js

```diff
@@ -4,34 +4,54 @@ const { QA_MENTION } = require('../config');
 module.exports = {
   name: Events.MessageCreate,
   async execute(message) {
-    if (message.author.bot) {
-      return;
-    }
+    try {
+      if (!message || !message.author || message.author.bot) {
```

Comment:
It could be shorten with optional chaining 
```suggestion
      if (!message?.author?.bot) {
```

Reply:
![image](https://github.com/user-attachments/assets/7b53be3d-0bec-4a5b-971e-aa0a3ec81ce9)
Move the conditional that is on line 18 to this block, seems similar